.gitignore

```diff
@@ -44,6 +44,9 @@ sparkler-job-**
 .cache-main
 .settings/
 
+# Netbeans-IDE specific
+*nbactions.xml
```

Comment:
should be a .gitignore on the sub directories too since there were still files in your commit with nbactions.xml